### DISCUSSION TITLE
Feature Idea: Play at beginning of Page

### DISCUSSION THREAD

**Discussion by opeologist** [SynthstromAudible/DelugeFirmware#946] [Ideas]
I just saw a video with an OXI One, where the user was editing a drum beat with multiple pages in the sequence. They seemed to lock the playing of the loop to just the page of the sequence they were editing, instead of it playing all 4 pages. I'd love if the Deluge can do something like this, as it's sometimes a pain to edit the middle of a drum sequence and have to play the beginning over again, when I just want to hear the section I'm working on to make sure it sounds like I expect. If this is possible already, please let me know how! Thank you!

Here's the video ref of the functionality: https://www.youtube.com/watch?v=vHXi__0HQrk&t=169s

The section the video starts at is bar 1 of 4, after he adjusts the bar (while looping just that bar), he moves onto bar 2, where he's editing it while just looping bar 2. Hopefully this video will explain the functionality properly!

**Comment by ok-reza** [comment]
already possible! hold <> encoder + PLAY, it will start playback on current screen.

**Reply by opeologist** [reply]
awesome, thanks so much for letting me know!
